main/src/mill/main/MainModule.scala

```diff
@@ -180,7 +182,7 @@ trait MainModule extends mill.Module {
       def rec(t: Task[_]): Seq[Segments] = {
         if (seen(t)) Nil // do nothing
         else t match {
-          case t: TargetImpl[_] if evaluator.rootModule.millInternal.targets.contains(t) =>
+          case t: mill.define.Target[_] if evaluator.rootModule.millInternal.targets.contains(t) =>
```

Comment:
This was a bug in https://github.com/com-lihaoyi/mill/pull/2402 that was surfaced when I ported doc snippets to example tests